pype/plugins/nukestudio/publish/collect_hierarchy_context.py

```diff
@@ -72,6 +74,24 @@ def process(self, context):
 
                 # and finding only hierarchical tag
                 if "hierarchy" in t_type.lower():
+                    match = next(
+                        (k for k, v in assets_shared.items()
+                         if (v["_clipIn"] == clip_in)
+                         and (v["_clipOut"] == clip_out)
+                         ), False)
```

Comment:
This is really unreadable...

Reply:
Understand, so do you suggest some other way?